code/lib/channels/src/index.ts

```diff
@@ -131,7 +131,8 @@ export class Channel {
     const listeners = this.listeners(event.type);
     if (listeners && listeners.length) {
       listeners.forEach((fn) => {
-        fn.apply(event, event.args);
+        // args should always be an array, but serverChannel seems to sometimes send a single value
+        fn.apply(event, Array.isArray(event.args) ? event.args : [event.args]);
```

Comment:
When does it send a single value? I don't love "sometimes" without knowing why.

Reply:
I fixed this